zap/src/main/java/org/parosproxy/paros/view/MainFooterPanel.java

```diff
@@ -109,7 +109,7 @@ public void initialise() {
         footerToolbarLeft.add(getAlertInfo());
 
         // Current Scans (Right)
```

Comment:
Might as well remove this (or update but my preference is avoid redundant comments).

Reply:
Tweaked